indico/modules/events/editing/templates/emails/comment_notification.txt

```diff
@@ -1,11 +1,17 @@
 {% extends 'events/editing/emails/base_email.txt' %}
 
-{% block subject -%}{% trans %}New comment{% endtrans %}{%- endblock %}
+{% set contribution=contribution.verbose_title %}
```

Comment:
I wouldn't override `contribution` like this, it might make debugging trickier... Either just pass the title to the template, or rename the var to something like `title`

Reply:
agreed, call it `title` here - it's also better for translators!